config/settings/base.py

```diff
@@ -279,3 +279,6 @@ def immutable_file_test(path, url):
 
 
 WHITENOISE_IMMUTABLE_FILE_TEST = immutable_file_test
+
+S3_FILE_BUCKET = env.str('DJANGO_S3_FILE_BUCKET')
+S3_FILE_PREFIX = env.str('DJANGO_S3_FILE_PREFIX', default='missioncontrol/filestorage/')
```

Comment:
There are some performance benefits on S3 of not having a path prefix, as files are sharded based on filename.

Reply:
Not so much!
https://aws.amazon.com/about-aws/whats-new/2018/07/amazon-s3-announces-increased-request-rate-performance/
> This S3 request rate performance increase removes any previous guidance to randomize object prefixes to achieve faster performance.

Reply:
oh awesome! Looks like they rolled that out last June.